integration/spark/app/src/test/java/io/openlineage/spark/agent/column/ColumnLineageWithTransformationTypesTest.java

```diff
@@ -167,11 +170,14 @@ void simpleQueryMultipleIndirect() {
     createTable("t1", "a;int", "b;int", "c;int");
     OpenLineage.ColumnLineageDatasetFacet facet =
         getFacetForQuery(
-            getSchemaFacet("a;int"), "SELECT a FROM t1 WHERE b > 1 GROUP BY a, c ORDER BY c");
-
+            getSchemaFacet("a;int", "c;int"),
+            "SELECT a, c FROM t1 WHERE b > 1 GROUP BY a, c ORDER BY c");
```

Comment:
Note: Here I added the `c` column in the `SELECT` because otherwise this wouldn't be a valid Hive query. I'm not sure why the Spark executor didn't complain about it previously.